app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
```

Comment:
The app should follow the design guidelines for the User login and registration page. Pages should use the same colors and fonts as defined in the [design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).

Reply:
Hey Sajjad. 🖖 Thank you for the review and feedback. 
Unfortunately, Proxima Nova font family is not free, it is paid font face. I found Lato font similar to Proxima Nova and used it. I hope that's okay. I fixed the authentication page styles.